### PR TITLE
Make the expand/collapse click area include the background

### LINKED ISSUE #133: Expand/collapse click area
### Expand/collapse click area
---

**Summary (short):**
It's currently just the text and arrow, but it should probably include the entire blue background as well.

---
**Description:**
Apply the fix to Ostron Bounties, event rewards lists, and Baro's inventory.

---
**Mockups, screenshots, evidence:**
Click the blue background of "Ostron Bounties" instead of the text or arrow.

---

(Check one)
- [ ] Issue
- [x] Suggestion

### PR DESCRIPTION
**Pull request by dsinn**
### Make the expand/collapse click area include the background
---
**Summary (short):**
Closes #133.

---
**Description:**
Basically what I suggested in the comments for the issue, except I needed to do some front-end tweaks afterwards.

---
**Fixes issue (include link):**
#133

---
**Mockups, screenshots, evidence:**

---

(Check one)
- [ ] Issue fix
- [x] Suggestion fulfillment

### CODE REVIEW

**Comment by dsinn** [comment]
I did the same thing for the Acolytes code-wise, but I'm not sure how to easily test them when they're not up.

**Comment by TobiTenno** [comment]
~~did you do it for baro? I haven't looked yet~~
looks like you did

File: assets/js/updaters.js [hunk 1]
```diff
@@ -269,11 +269,11 @@ function updateVoidTraderInventory() {
       /* eslint-disable prefer-template */
       const inventoryString = `${'<div class="panel panel-primary voidTraderInventory" ' +
                 'style="margin-left:5%; margin-right:5%" ' +
-                'id="'}${worldState.voidTrader.id}">\n<div class="panel-heading">\n` +
+                'id="'}${worldState.voidTrader.id}">\n` +
+                '<a href="#voidTraderInventoryPanel" data-toggle="collapse" class="panel-heading show">\n' +
```
Comment (TobiTenno) [inline assets/js/updaters.js]: could you add a text-center to the anchor classes here, that way it's consistent with the other panels?

File: assets/js/updaters.js [hunk 2]
```diff
@@ -269,11 +269,11 @@ function updateVoidTraderInventory() {
...
                 `<h3 class="panel-title">${worldState.voidTrader.character} Inventory` +
-                '<a href="#voidTraderInventoryPanel" data-toggle="collapse">' +
-                '<span class="glyphicon glyphicon-triangle-bottom pull-right"></span></a></h3>\n' +
-                '</div>\n' +
+                '<span class="glyphicon glyphicon-triangle-bottom pull-right"></span></h3>\n' +
+                '</a>\n' +
                 '<div class="panel-body collapse" id="voidTraderInventoryPanel">\n' +
                 '<table class="table table-striped table-hover ">\n' +
```
Comment (TobiTenno) [inline assets/js/updaters.js:278]: looks like the headings on the table are a tad misalinged on mobile resolutions... If you see a fix for it, could you add that?

**Review by TobiTenno — CHANGES_REQUESTED** [review]

**Comment by dsinn** [comment]
Yeah, I wrote this a while ago but specifically waited for Baro to appear because I'm not aware of a convenient way to inject data for testing.

**Review by TobiTenno — APPROVED** [review]

**Pull request merged**